docs/settings.rst

```diff
@@ -141,6 +141,25 @@ Language Detection
     datetime.datetime(2015, 12, 4, 0, 0)
 
 
+Default Languages
++++++++++++++++++
+
+``DEFAULT_LANGUAGES``: It is a ``list`` of language codes in ISO 639 which is used as default 
+languages to be used for translation and parsing. eg. "en":
+
+    >>> from dateparser import parse
+    >>> parse(settings={'3 de marzo de 2020','DEFAULT_LANGUAGES': ["es"]})
```

Comment:
Hi @gavishpoddar, this is wrong it should probably be like this:

```suggestion
    >>> parse('3 de marzo de 2020', settings={'DEFAULT_LANGUAGES': ["es"]})
```

Reply:
Done 👍🏻